tutorials/self-signed-docker-registry-on-gke.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Using a self-signed Docker Registry on GKE
+description: Using a self-signed Docker Registry on GKE.
+author: samos123
+tags: Kubernetes Engine, GKE, Docker Registry
+date_published: 2020-01-28
+---
+
+This tutorial shows how to use a Docker registry that was signed by a 
+non-trusted CA with Google Kubernetes Engine (GKE). Let's assume your company
+has a private CA that's not trusted by default.
+All internal services such as on-prem docker registry have their certs signed
+by this private CA. This would cause GKE to prevent pulling from the on-prem
+registry since it's not trusted. In this guide we assume the private CA cert is
+stored in a file called `myCA.pem`.
+
+The solution is to import the CA into the system bundle and restart docker.
+However on GKE, COS is used which makes this harder. The following steps
+are needed on GKE:
+1. Create a docker image that that has `myCA.pem`, copies the `myCA.pem` 
+   into `/mnt/etc/ssl/certs/`, runs `update-ca-certificates` and restart
+   docker daemon.
+2. Create a daemonset that runs this docker image on every node and mount
+   the host `/etc` directory to the containers `/mnt/etc` directory
+
+## Before you begin
+You will need the following resources:
+* A Docker registry that was signed by an untrusted CA
```

Comment:
I would recommend you either create a private docker registry as part of the tutorial or use something like dockerhub to simulate this.